C-Plus-Plus/cryptography/Modified_Caeser_Cipher.cpp

```diff
@@ -0,0 +1,63 @@
+//Modified Caeser cipher
+
+/*
+It is a type of susbtitution cipher which encrypts the plain text 
+message into a cipher text by simply replacing each letter
+of a given text by some places down the alphabet.
+
+
+Example:
+Text : TESSERACTCODING
+Shift: 4
+Cipher: XIWWIVEGXGSHMRK
+
+Text : NEOALGO
+Shift: 2
+Cipher: PGQCNIQ
+
+Time Complexity: O(n)
+Space Complexity: O(n)
```

Comment:
Shift Examples and Complexities to end of the Code

Reply:
Done. Thank you for suggesting the changes.